.github/workflows/publish.yaml

```diff
@@ -77,6 +77,16 @@ jobs:
           npm run build:env -- $BUILD_PARAMS
 
       - name: Publish if it was clicked manually
+        id: publish
         if: ${{ github.event_name == 'repository_dispatch' || (github.event.inputs.buildImage == 'true' && success()) }}
         run: |
           npm run publish:env
+
+      - name: Trigger Bee-js PR creation
+        if: ${{ github.event_name == 'repository_dispatch' || (github.event.inputs.buildImage == 'true' && success()) }}
```

Comment:
I think it shouldn't trigger anything if the event happened by manual action.

Reply:
True! Great catch!